Wikipedia/Code/Controllers/WikidataDescriptionController.swift

```diff
@@ -49,4 +49,23 @@ class WikidataDescriptionController: ArticleDescriptionControlling {
     func learnMoreViewControllerWithTheme(_ theme: Theme) -> UIViewController? {
         return DescriptionHelpViewController.init(theme: theme)
     }
+    
+    func warningTypesForDescription(_ description: String?) -> ArticleDescriptionWarningTypes {
+        
+        var warningTypes: ArticleDescriptionWarningTypes = []
+        
+        if descriptionIsTooLong(description) {
+            warningTypes.insert(.length)
+        }
+
+        guard let firstCharacter = description?.first,
+              firstCharacter.isLetter,
+              firstCharacter.isUppercase else {
+            return warningTypes
+        }
+        
+        warningTypes.insert(.casing)
```

Comment:
To me, it's more readable if something like this. Personal preference though.

```suggestion
        if let firstCharacter = description?.first,
              firstCharacter.isLetter,
              firstCharacter.isUppercase {
              warningTypes.insert(.casing)
        }
```

Reply:
@mcleinman done!